native/src/routes/SprungbrettOfferContainer.tsx

```diff
@@ -36,24 +29,13 @@ const SprungbrettOfferContainer = ({ route, navigation }: SprungbrettOfferContai
   const shareUrl = urlFromRouteInformation({ route: SPRUNGBRETT_OFFER_ROUTE, languageCode, cityCode })
   useHeader({ navigation, route, availableLanguages, data, shareUrl })
 
-  const navigateToFeedback = (isPositiveFeedback: boolean) => {
-    createNavigateToFeedbackModal(navigation)({
-      routeType: OFFERS_ROUTE,
-      cityCode,
-      slug: SPRUNGBRETT_OFFER_ROUTE,
```

Comment:
I think this also breaks with the changes

Reply:
added this to the header to define correct slug for this route